### PR TITLE
Merge back changes since April

### PR DESCRIPTION
**Pull request by martinwe-adfinis**
Time to keep things up to date!

### CODE REVIEW

File: etc/sh/profile.d/99-private.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Load private profile.d snippets
```
Comment (ayekat) [inline etc/sh/profile.d/99-private.sh]: This used to be already done with `etc/sh/profile` (now moved into `etc/sh/profile.d/90-private.sh` in cea0a059e4f05ee47c53628ed8e8e219c605923b). Was that somehow missed?

Anyway, this seems unnecessary to me.

**Comment by ayekat** [comment]
On the scrlock change, I had already pointed this out in the previous MR (#39), but on second thought, I think it's fine to have this here for now until we've (slowly) cleaned up the packaging mess around karuiwm.

File: etc/sh/profile.d/99-private.sh (hunk 1 above)
Comment (martinwe-adfinis) [reply etc/sh/profile.d/99-private.sh]: Somehow missed that completely…. I've removed the commit.

**Comment by ayekat** [comment]
Works for me, merging.

**--edit**: Missed that I've pushed something earlier today. Need to rebase >.<

**Comment by martinwe-adfinis** [comment]
Rebased.

**Pull request merged**